dash/dash.py

```diff
@@ -239,10 +239,14 @@ def __init__(
 
         # We have 3 cases: server is either True (we create the server), False
         # (defer server creation) or a Flask app instance (we use their server)
-        if isinstance(server, bool):
-            self.server = Flask(name) if server else None
-        elif isinstance(server, Flask):
+        if isinstance(server, Flask):
             self.server = server
+            # GH 771
```

Comment:
I don't think we need this comment - that's what `git blame` is for.

Reply:
Sure thing I will remove it, each project has its own rules 😄